src/sentry/replays/blueprints/api.md

```diff
@@ -608,3 +608,103 @@ Retrieve a collection of click events associated with a replay.
     ]
   }
   ```
+
+## Replays Viewed Status [/organizations/<organization_slug>/replays-viewed/]
+
+- Parameters
+
+  - project (optional, string)
+  - id (required, string) - The id(s) of the replay to retrieve. To specify multiple ids, use the format `?id=a&id=b ...`. A maximum of 100 id's can be specified.
+
+### Browse Replays Viewed Status [GET]
+
+Returns a collection of replays and their viewed state. The user is targeted based on the authorization context. Replay-ids are specified in the URL parameters.
```

Comment:
This'll be a good place to re-use our existing `useAggregatedQueryKeys` javascript function to fetch results in batches.